### PR TITLE
Add upgrade doc to v1.12

### PR DESCRIPTION
**Pull request by allenxu404**
Thank you for contributing to Velero!

# Please add a summary of your change
- Add upgrade-to-1.12 doc

# Does your change fix a particular issue?

# Please indicate you've done the following:

- [ ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Review by qiuming-best — APPROVED** [review]

File: site/content/docs/main/upgrade-to-1.12.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+title: "Upgrading to Velero 1.12"
 layout: docs
 ---
 
 ## Prerequisites
 
-- Velero [v1.10.x][5] installed.
+- Velero [v1.11.x][5] installed.
 
 If you're not yet running at least Velero v1.6, see the following:
```
Comment (mateusoliveira43) [inline site/content/docs/main/upgrade-to-1.12.md:10]: change to v1.7 here?

**Review by sseago — APPROVED** [review]

File: site/content/docs/main/upgrade-to-1.12.md [hunk 2]
```diff
@@ -77,41 +77,41 @@ Before upgrading, check the [Velero compatibility matrix](https://github.com/vmw
...
-3. Update the container image used by the Velero deployment, plugin and, optionally, the node agent daemon set:
+1. Update the container image used by the Velero deployment, plugin and, optionally, the node agent daemon set:
     ```bash
    # set the container and image of the init container for plugin accordingly,
    # if you are using other plugin
     kubectl set image deployment/velero \
-        velero=velero/velero:v1.11.0 \
-        velero-plugin-for-aws=velero/velero-plugin-for-aws:v1.7.0 \
+        velero=velero/velero:v1.12.0 \
+        velero-plugin-for-aws=velero/velero-plugin-for-aws:v1.8.0 \
```
[the remark this answers is not in the document]
Comment (kaovilai) [reply site/content/docs/main/upgrade-to-1.12.md:96]: https://hub.docker.com/r/velero/velero-plugin-for-aws/tags

[the remark this answers is not in the document]
Comment (kaovilai) [reply site/content/docs/main/upgrade-to-1.12.md:96]: from discussions I've received with a maintainer, bumping Y version in plugins is typical.

**Review by kaovilai — APPROVED** [review]

**Review by qiuming-best — APPROVED** [review]

**Pull request merged**